.rubocop.yml

```diff
@@ -58,9 +60,6 @@ Style/Documentation:
 Style/ExponentialNotation:
   Enabled: true
 
-Style/OpenStructUse:
-  Enabled: false
-
```

Comment:
This is the only _explicit_ rule from newer RuboCop releases that the older release, introduced by this PR, didn't support.